tutorials/xr/deploying_to_android.rst

```diff
@@ -78,28 +61,20 @@ Next change the name of the export preset for the device you're setting this up
 And enable **Use Gradle Build**.
 If you want to use one-click deploy (described below), ensure that **Runnable** is enabled.
 
-If the vendors plugins were installed correctly you should find entries for the different headsets, select the entry for meta:
+If the vendors plugins were installed correctly you should find entries for the
+different headsets under **XR Features**. Change the **XR Mode** to **OpenXR**, then
+select the entry for your headset if you see one. If you don't see one enable the
+Khronos plugin. It's the official Khronos loader, in due time all headsets should
+work with this loader. At the moment this loader has been tested with the Magic Leap
+2 and standalone HTC headsets.
 
 .. image:: img/android_meta_quest.webp
 
-Also change the **XR Mode** to **OpenXR**.
 
-Scroll to the bottom of the list and you'll find additional XR feature sections, currently only **Meta XR Features** is available.
-The hand tracking and passthrough settings here currently only work for the Meta Quest and you will need to select the appropriate settings if you wish to use these features.
 
-.. image:: img/xr_export_features.webp
-
-Now you can repeat the same process for the other devices.
-
-.. note::
-    There are separate loaders for the Meta Quest, Pico and Lynx R1 headsets.
-
-    The fourth option is the official Khronos loader, in due time all headsets should work with this loader.
-    At the moment this loader has been tested with the Magic Leap 2 and standalone HTC headsets.
-
-.. warning::
-    While the Mobile Vulkan renderer has many optimizations targeted at mobile devices, we're still working out the kinks.
-    It is highly advisable to use the compatibility renderer (OpenGL) for the time being when targeting Android based XR devices.
+Scroll to the bottom of the list and you'll find additional XR feature sections,
+currently only **Meta XR Features** and **Pico XR Features** are available.
```

Comment:
There should be a Khronos features section, but I think a bug must have recently been introduced in the filtering because it's missing in the editor (it's still there in the code).

Reply:
I have to ask a stupid question then, is this something you expect to be fixed by the time Godot 4.4 releases? And I know that's hard to guess without knowing what the issue is, but I'd rather not have the docs reference settings that might not exist at the time of that release. Also I've created an issue in the plugin repository for this https://github.com/GodotVR/godot_openxr_vendors/issues/256